qa/supply-chain/audits.toml

```diff
@@ -24,6 +24,16 @@ who = "Jack Grigg <jack@electriccoin.co>"
 criteria = ["safe-to-deploy", "crypto-reviewed"]
 delta = "0.5.1 -> 0.5.2"
 
+[[audits.allocator-api2]]
+who = "Daira Emma Hopwood <daira@jacaranda.org>"
+criteria = "safe-to-deploy"
+version = "0.2.14"
+notes = """
+This is a workaround for https://doc.rust-lang.org/unstable-book/library-features/allocator-api.html being unstable.
+It copies the unstable implementation, and inherently requires a lot of unsafe code, which was infeasible to review completely.
+However, safety requirements appear to be well-documented.
+"""
+
```

Comment:
I am not comfortable with including this audit, as it does not meet the [`safe-to-deploy` standard](https://mozilla.github.io/cargo-vet/built-in-criteria.html#safe-to-deploy) (by virtue of the unsafe blocks not having been fully reviewed). We might want to introduce a mid-point standard of review (that implies `safe-to-run` and is implied by `safe-to-deploy`) that can be used when we don't have time for a full `unsafe` / powerful capability audit. But for now we should remove this and replace it with an exemption.
```suggestion
```